MANIFEST.in

```diff
@@ -1 +0,0 @@
-prune profiling
```

Comment:
This isn't being pruned in the new Hatchling configuration, the sdist is over twice as big because of this (!)

```
❯ lexa setuptools-dist/
.rw-rw-r-- 160k ichard26 ichard26  5 Sep 22:10 black-22.8.1.dev4+g383b228-py3-none-any.whl
.rw-rw-r-- 578k ichard26 ichard26  5 Sep 22:10 black-22.8.1.dev4+g383b228.tar.gz

❯ lexa hatch-dist/
.rw------- 164k ichard26 ichard26  5 Sep 22:07 black-22.8.1.dev5+g7a3ba27-py3-none-any.whl
.rw------- 1.2M ichard26 ichard26  5 Sep 22:07 black-22.8.1.dev5+g7a3ba27.tar.gz
```